### ISSUE TITLE
rpm not working

### ISSUE THREAD

**Issue by eldryoth** [pagespeed/ngx_pagespeed#824]
Hi,

I am trying to build some rpms for my repository mainly to help out a friend, and perhaps other people can use it also however I have some trouble to get it to work correctly.

Everything is compiling correctly no errors during the rpm creation or that I am able to see in the output. 
I can also install it correctly without issue also nginx runs perfectly fine as long as pagespeed is disable, once I enable pagespeed by adding the following to my nginx.conf:

pagespeed On;
pagespeed FileCachePath "/var/cache/nginx/pagespeed_cache/"; (nginx has correcty permission here to write)

Nginx won't start any workers it only has the main process that keeps restarting with the following error in my error logs.

2014/10/11 10:34:03 [alert] 621#0: worker process 23244 exited on signal 6
2014/10/11 10:34:03 [alert] 23245#0: [ngx_pagespeed 1.9.32.1-4238] [1011/103403:FATAL:ngx_rewrite_driver_factory.cc(205)] Check failed: ok. Unable to start scheduler thread
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x4c8e6a]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x495fd0]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x4c5d82]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x4a278f]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x49a2b5]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x42b426]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x42b997]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x42a4d4]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x42cac2]
nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf() [0x40e14f]
/lib64/libc.so.6(__libc_start_main+0xf5) [0x7f3992568b45]

Has anyone seen this error before and is there perhaps something I did wrong?
Spec file I created basically the one nginx provides: https://raw.githubusercontent.com/sbagmeijer/ulyaoth/master/Repository/ulyaoth-nginx/SPECS/ulyaoth-nginx-pagespeed.spec

Basically all I do is add this:
--add-module=/etc/nginx/modules/pagespeed \

And create the caching directory automatically.

Thank you in advanced.

**Comment by sbagmeijer** [comment]
Thank you kyl191 I will give that a try to see if it works.

**Comment by sbagmeijer** [comment]
Solved this problem now by following the Fedora guide:
https://fedoraproject.org/wiki/PackagingDrafts/SELinux#Adding_an_existing_SELinux_policy_to_an_application

And as they suggested I created a separate package for the selinux policy so people can choose if they wish to use it or not as the advice was on that website.

Policy:
https://github.com/sbagmeijer/ulyaoth/blob/master/Repository/ulyaoth-nginx-pagespeed/Selinux/ulyaoth-nginx-pagespeed.te

Spec file:
https://github.com/sbagmeijer/ulyaoth/blob/master/Repository/ulyaoth-nginx-pagespeed/SPECS/ulyaoth-nginx-pagespeed-selinux.spec

It works all great! :+1:
